src/test/java/uk/gov/ons/ctp/response/sample/ingest/CsvIngesterBusinessTest.java

```diff
@@ -59,6 +67,59 @@
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
+  private static final String SAMPLEUNITREF = "1234567890";
```

Comment:
None of these variables are used

Reply:
No idea why these are here - they've been there since the beginning.  I'll remove them